requirements.txt

```diff
@@ -31,6 +31,7 @@ psutil
 protobuf==3.20.1 # https://github.com/databrickslabs/dbx/issues/257
 experiment_impact_tracker
 gpustat
+rich~=12.4.4
```

Comment:
This dependency is specifically for console messages when using the config object in a notebook. It specifically allows for more robust messages after enabling and disabling features. It would be a little bit tricky to factor out into a separate PR like we discussed in standup because it would require the full config object lands before so it would be better as a followup PR. That being said, it is a very small change and if we are not on board with it, I'm ok just removing it altogether since it's a small UX improvement on the SDK side.